src/libcollections/btree/node.rs

```diff
@@ -1440,11 +1320,11 @@ unsafe fn move_kv<K, V>(
 }
 
 // Source and destination must have the same height.
-unsafe fn move_edges<K, V>(
-    mut source: NodeRef<marker::Mut, K, V, marker::Internal>, source_offset: usize,
-    mut dest: NodeRef<marker::Mut, K, V, marker::Internal>, dest_offset: usize,
-    count: usize)
-{
+unsafe fn move_edges<K, V>(mut source: NodeRef<marker::Mut, K, V, marker::Internal>,
+                           source_offset: usize,
+                           mut dest: NodeRef<marker::Mut, K, V, marker::Internal>,
+                           dest_offset: usize,
+                           count: usize) {
```

Comment:
Ditto - keeping the division of lines that was here previously makes things clearer, I think.